config.json

```
@@ -1143,6 +1143,62 @@
     ]
   },
   "concepts": [],
-  "key_features": [],
-  "tags": []
+  "key_features": [
+    {
+      "title": "Evolving",
+      "content": "Typescript is a modern and constantly evolving language supported by Microsoft."
+    },
+    {
+      "icon": "features-cross-platform",
+      "title": "Widely used",
+      "content": "Typescript is used by many frontend frameworks, and can be used in Javascript based ones as well."
```

```suggestion
      "content": "TypeScript is used by many libraries and frameworks, and integrates with JavaScript code as well."
```

I don't think TypeScript is focused on frontend. Case-in-point. The TypeScript Analyzer, Representer, JavaScript Analyzer, Representer, and the static-analysis tools on exercism are all written in TypeScript 🥇

---

That's amazing :D, I though that it was also used in deno and node if configured correctly, and I supposed that you could do "anything" if you used it well, but lacked actual examples other than theory